### PR TITLE
[PPP-4776] CE fix for quartz upgrade.

### PR DESCRIPTION
**Pull request by tkafalas**

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 9m 24s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -pl assembly
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Review by ddiroma — APPROVED** [review]

File: assembly/src/main/quartz/quartz.properties [hunk 1]
```diff
@@ -297,10 +297,10 @@ org.quartz.jobStore.class = org.quartz.impl.jdbcjobstore.JobStoreTX
 #_replace_jobstore_properties
 
 org.quartz.jobStore.misfireThreshold = 60000
-org.quartz.jobStore.driverDelegateClass = org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
+org.quartz.jobStore.driverDelegateClass = org.quartz.impl.jdbcjobstore.HSQLDBDelegate
```
Comment (peterrinehart) [inline assembly/src/main/quartz/quartz.properties:300]: Was this just a forgotten change from before?

**Review by peterrinehart — CHANGES_REQUESTED** [review]

File: assembly/src/main/quartz/quartz.properties (hunk 1 above)
Comment (tkafalas) [reply assembly/src/main/quartz/quartz.properties:300]: Apparently, yes.  I figured it's better than misstating it though the SQL probably doesn't change for what we use it for

**Pull request merged**